curriculum/challenges/english/25-front-end-development/lab-availability-table/66b36358ed4f261d64840c24.md

```diff
@@ -291,11 +291,46 @@ You should use two color-stops (expressed in percentage) to make the transition
 ```js
 const legendGradient = new __helpers.CSSHelp(document).getStyle('#legend-gradient');
 assert.exists(legendGradient);
-const matched = legendGradient.backgroundImage.match(/var\(\s*--color[0-5]\s*\)\s+\d+%(\s+\d+%)?/g);
-assert.lengthOf(matched, 6);
-matched.forEach((arg, i) => {
-  if (i !== 0 && i !== 5) assert.lengthOf(arg.match(/%/g), 2);
-})
+
+const doubleStop = legendGradient.backgroundImage.match(/var\(\s*--color[0-5]\s*\)\s+\d+(\.\d+)?%/g) || [];
+const singleStop = legendGradient.backgroundImage.match(/var\(\s*--color[0-5]\s*\)\s+\d+%(\s+\d+%)?/g) || [];
```

Comment:
the names maybe are inverted?

the multistop one is the one with two percentages for each color